plugins/android-boot/README.md

```diff
@@ -0,0 +1,51 @@
+# Android Bootloaders
+
+## Introduction
+
+This plugin is used to update hardware that use partitions to store their firmware on.
```

Comment:
I think it would be good here to clarify why and how this is different from fastboot